### ISSUE TITLE
Can not start app with version 0.1.12

### ISSUE THREAD

**Issue by Anduin2017** [BoxFactura/pulltorefresh.js#37]
My app is here:
[https://github.com/AiursoftWeb/Kahla.App](https://github.com/AiursoftWeb/Kahla.App)

Today I have just updated all dependencies to the latest version and my app just could run.

Every operation with pulltorefresh package returns an error as:

```
PullToRefresh.destroyAll is not a function
PullToRefresh.init is not a function
```

I guess the problem happens from the package itself.

My code is here

https://github.com/AiursoftWeb/Kahla.App/blob/master/src/app/Controllers/conversations.component.ts#L27

**Comment by pateketrueke** [comment]
My apologies, I rebuilt the library not using an UMD format, I fixed and released `v0.1.13` right now!

**Comment by Anduin2017** [comment]
Thanks. Now my app works now!
